docker-compose.yml

```diff
@@ -15,14 +15,25 @@
 version: '3.1'
 services:
   web:
-    build: .
+    build: 
+      context: .
+      args:
```

Comment:
SUGGESTION: Based on previous suggestion, if you have these set in the .env file and pick them up as ENV in the Dockerfile, you don't need to pass these